src/ucp/proto/proto_multi.c

```diff
@@ -191,12 +150,12 @@ ucs_status_t ucp_proto_multi_init(const ucp_proto_multi_init_params_t *params,
         /* Make sure fragment is not zero */
         ucs_assert(max_frag > 0);
 
-        lpriv->max_frag = max_frag;
-        perf.max_frag  += max_frag;
+        lpriv->max_frag    = max_frag;
```

Comment:
minor: align by =, like in the rest of the code

Reply:
Done, thanks